README.md

```diff
@@ -0,0 +1,117 @@
+CSS Foldable Display polyfills
```

Comment:
I'm sure CSSWG will bikeshed any name we have today and it'll change a 1000 times, for sake of simplicity, do you think calling it `CSS Spanning Media Feature Polyfill` is more simple & direct to what this polyfill does?